link-rest/src/main/java/com/nhl/link/rest/meta/LrAttribute.java

```diff
@@ -9,12 +10,20 @@
  */
 public interface LrAttribute {
 
+	/**
+	 * @since 1.12
+     */
 	String getName();
 
 	/**
 	 * @since 1.24
 	 */
 	Class<?> getType();
 
+	/**
+	 * @since 1.12
+     */
 	ASTPath getPathExp();
+
+	Object extractValue(JsonNode node); // TODO: move this to a separate place?
```

Comment:
In fact instead of creating a new facility, we can add these methods to the existing IJsonValueConverterFactory